tests/modules/tnotuniquename2.nim

```diff
@@ -1,7 +1,7 @@
 discard """
   errormsg: "module names need to be unique per Nimble package"
   file: "tnotuniquename/mnotuniquename.nim"
-  disabled: "true"
+    disabled: "true"
```

Comment:
Disabled around 2019-04 in a commit that references #11064
Seems to be testing removed behaviour. Probably should be removed.